[CI] ArchiveGenerateInitialCredentialsTests test20NoAutoGenerationWhenAutoConfigurationDisabled failing

**Build Scans:**
- [elasticsearch-periodic-packaging #4501 / rhel-7_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/dbvl7wqvpuvlw)
- [elasticsearch-periodic-packaging #4501 / ubuntu-1804_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/uqo763n56eu6g)
- [elasticsearch-periodic-packaging #4501 / sles-12_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/lpwotwek4i5g4)
- [elasticsearch-periodic-packaging #4501 / oraclelinux-7_packaging-tests-unix](https://gradle-enterprise.elastic.co/s/mqn6oep26mxtc)
- [elasticsearch-pull-request #40830 / ubuntu-2004_destructivedistrotest.archives_packaging-tests-unix-sample](https://gradle-enterprise.elastic.co/s/jxbkp2i73j53q)
- [elasticsearch-pull-request #40830 / rhel-8_destructivedistrotest.archives_packaging-tests-unix-sample](https://gradle-enterprise.elastic.co/s/wx4ne7hr35w56)
- [elasticsearch-pull-request #40766 / ubuntu-2004_destructivedistrotest.archives_packaging-tests-unix-sample](https://gradle-enterprise.elastic.co/s/eit4lu44yjp7u)
- [elasticsearch-pull-request #40765 / ubuntu-2004_destructivedistrotest.archives_packaging-tests-unix-sample](https://gradle-enterprise.elastic.co/s/tsobzcpdwhgge)
- [elasticsearch-pull-request #40766 / rhel-8_destructivedistrotest.archives_packaging-tests-unix-sample](https://gradle-enterprise.elastic.co/s/fvv5lock2k7vq)
- [elasticsearch-pull-request #40765 / rhel-8_destructivedistrotest.archives_packaging-tests-unix-sample](https://gradle-enterprise.elastic.co/s/kix6ymeuc2kjy)

**Reproduction Line:**
```
null
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.packaging.test.ArchiveGenerateInitialCredentialsTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test20NoAutoGenerationWhenAutoConfigurationDisabled),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Startup command should succeed. Stderr: [exitCode = [1] stdout = [spawn -ignore HUP sudo -E -u elasticsearch /tmp/elasticsearch/bin/elasticsearch -p /tmp/elasticsearch/elasticsearch.pid -v
Skipping security auto configuration because [xpack.security.autoconfiguration.enabled] is false
CompileCommand: dontinline java/lang/invoke/MethodHandle.setAsTypeCache bool dontinline = true
CompileCommand: dontinline java/lang/invoke/MethodHandle.asTypeUncached bool dontinline = true
[2024-11-11T21:11:00,057][INFO ][o.e.n.NativeAccess       ] [bk-agent-prod-gcp-1731359030615088306] Using native vector library; to disable start with -Dorg.elasticsearch.nativeaccess.enableVectorLibrary=false
[2024-11-11T21:11:00,106][INFO ][o.e.n.NativeAccess       ] [bk-agent-prod-gcp-1731359030615088306] Using [jdk] native provider and native methods for [Linux]
[2024-11-11T21:11:00,234][INFO ][o.a.l.i.v.PanamaVectorizationProvider] [bk-agent-prod-gcp-1731359030615088306] Java vector incubator API enabled; uses preferredBitSize=256; FMA enabled
[2024-11-11T21:11:01,030][WARN ][x-pack-ml-controller-stderr] [bk-agent-prod-gcp-1731359030615088306] /tmp/elasticsearch/modules/x-pack-ml/platform/linux-x86_64/bin/controller: error while loading shared libraries: libzstd.so.1: cannot open shared object file: No such file or directory
[2024-11-11T21:11:02,480][INFO ][o.e.n.Node               ] [bk-agent-prod-gcp-1731359030615088306] version[9.0.0-SNAPSHOT], pid[11819], build[tar/20543579024175b76a621a67dfb4e8924d240263/2024-11-11T21:06:54.300091999Z], OS[Linux/3.10.0-1160.119.1.el7.x86_64/amd64], JVM[Oracle Corporation/OpenJDK 64-Bit Server VM/23/23+37-2369]
[2024-11-11T21:11:02,482][INFO ][o.e.n.Node               ] [bk-agent-prod-gcp-1731359030615088306] JVM home [/tmp/elasticsearch/jdk], using bundled JDK [true]
[2024-11-11T21:11:02,482][INFO ][o.e.n.Node               ] [bk-agent-prod-gcp-1731359030615088306] JVM arguments [-Des.networkaddress.cache.ttl=60, -Des.networkaddress.cache.negative.ttl=10, -XX:+AlwaysPreTouch, -Xss1m, -Djava.awt.headless=true, -Dfile.encoding=UTF-8, -Djna.nosys=true, -XX:-OmitStackTraceInFastThrow, -Dio.netty.noUnsafe=true, -Dio.netty.noKeySetOptimization=true, -Dio.netty.recycler.maxCapacityPerThread=0, -Dlog4j.shutdownHookEnabled=false, -Dlog4j2.disable.jmx=true, -Dlog4j2.formatMsgNoLookups=true, -Djava.locale.providers=CLDR, -Des.distribution.type=tar, --enable-native-access=org.elasticsearch.nativeaccess,org.apache.lucene.core, -XX:ReplayDataFile=logs/replay_pid%p.log, -Djava.security.manager=allow, -XX:+UseG1GC, -Djava.io.tmpdir=/tmp/elasticsearch-7487697842370166026, --add-modules=jdk.incubator.vector, -XX:CompileCommand=dontinline,java/lang/invoke/MethodHandle.setAsTypeCache, -XX:CompileCommand=dontinline,java/lang/invoke/MethodHandle.asTypeUncached, -Dorg.apache.lucene.store.defaultReadAdvice=normal, -XX:+HeapDumpOnOutOfMemoryError, -XX:+ExitOnOutOfMemoryError, -XX:HeapDumpPath=data, -XX:ErrorFile=logs/hs_err_pid%p.log, -Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,level,pid,tags:filecount=32,filesize=64m, -Xmx1g, -Xms1g, -XX:MaxDirectMemorySize=536870912, -XX:G1HeapRegionSize=4m, -XX:InitiatingHeapOccupancyPercent=30, -XX:G1ReservePercent=15, --module-path=/tmp/elasticsearch/lib, --add-modules=jdk.net, --add-modules=ALL-MODULE-PATH, -Djdk.module.main=org.elasticsearch.server]
[2024-11-11T21:11:02,484][INFO ][o.e.n.Node               ] [bk-agent-prod-gcp-1731359030615088306] Default Locale [en_US]
[2024-11-11T21:11:02,484][WARN ][o.e.n.Node               ] [bk-agent-prod-gcp-1731359030615088306] version [9.0.0-SNAPSHOT] is a pre-release version of Elasticsearch and is not suitable for production
[2024-11-11T21:11:03,970][INFO ][o.e.c.u.FeatureFlag      ] [bk-agent-prod-gcp-1731359030615088306] The current build is a snapshot, feature flag [esql_date_nanos] is enabled
[2024-11-11T21:11:03,971][INFO ][o.e.c.u.FeatureFlag      ] [bk-agent-prod-gcp-1731359030615088306] The current build is a snapshot, feature flag [esql_semantic_text] is enabled
[2024-11-11T21:11:05,785][INFO ][o.e.c.u.FeatureFlag      ] [bk-agent-prod-gcp-1731359030615088306] The current build is a snapshot, feature flag [esql_inlinestats] is enabled
[2024-11-11T21:11:09,246][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [repository-url]
[2024-11-11T21:11:09,247][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [rest-root]
[2024-11-11T21:11:09,247][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-core]
[2024-11-11T21:11:09,247][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-redact]
[2024-11-11T21:11:09,248][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [ingest-user-agent]
[2024-11-11T21:11:09,248][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-async-search]
[2024-11-11T21:11:09,248][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [test-error-query]
[2024-11-11T21:11:09,249][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-monitoring]
[2024-11-11T21:11:09,249][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [repository-s3]
[2024-11-11T21:11:09,249][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-analytics]
[2024-11-11T21:11:09,250][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-esql-core]
[2024-11-11T21:11:09,250][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-ent-search]
[2024-11-11T21:11:09,250][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-autoscaling]
[2024-11-11T21:11:09,251][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [lang-painless]
[2024-11-11T21:11:09,251][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-ml]
[2024-11-11T21:11:09,251][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [test-die-with-dignity]
[2024-11-11T21:11:09,251][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [lang-mustache]
[2024-11-11T21:11:09,252][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [legacy-geo]
[2024-11-11T21:11:09,252][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [logsdb]
[2024-11-11T21:11:09,252][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-ql]
[2024-11-11T21:11:09,252][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [rank-rrf]
[2024-11-11T21:11:09,253][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [analysis-common]
[2024-11-11T21:11:09,253][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [health-shards-availability]
[2024-11-11T21:11:09,253][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [transport-netty4]
[2024-11-11T21:11:09,253][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [aggregations]
[2024-11-11T21:11:09,254][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [ingest-common]
[2024-11-11T21:11:09,255][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [frozen-indices]
[2024-11-11T21:11:09,255][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-identity-provider]
[2024-11-11T21:11:09,255][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-shutdown]
[2024-11-11T21:11:09,256][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-text-structure]
[2024-11-11T21:11:09,256][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [snapshot-repo-test-kit]
[2024-11-11T21:11:09,256][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [test-delayed-aggs]
[2024-11-11T21:11:09,257][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [ml-package-loader]
[2024-11-11T21:11:09,257][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [kibana]
[2024-11-11T21:11:09,257][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [test-esql-heap-attack]
[2024-11-11T21:11:09,257][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [constant-keyword]
[2024-11-11T21:11:09,258][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-logstash]
[2024-11-11T21:11:09,258][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-ccr]
[2024-11-11T21:11:09,258][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-graph]
[2024-11-11T21:11:09,259][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-esql]
[2024-11-11T21:11:09,259][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [parent-join]
[2024-11-11T21:11:09,260][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-enrich]
[2024-11-11T21:11:09,260][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [counted-keyword]
[2024-11-11T21:11:09,260][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [repositories-metering-api]
[2024-11-11T21:11:09,260][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [transform]
[2024-11-11T21:11:09,261][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [repository-azure]
[2024-11-11T21:11:09,261][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [dot-prefix-validation]
[2024-11-11T21:11:09,261][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [repository-gcs]
[2024-11-11T21:11:09,261][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-otel-data]
[2024-11-11T21:11:09,262][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [spatial]
[2024-11-11T21:11:09,262][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [apm]
[2024-11-11T21:11:09,264][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [mapper-extras]
[2024-11-11T21:11:09,264][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [mapper-version]
[2024-11-11T21:11:09,265][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [test-jvm-crash]
[2024-11-11T21:11:09,265][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-rollup]
[2024-11-11T21:11:09,265][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [percolator]
[2024-11-11T21:11:09,265][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [data-streams]
[2024-11-11T21:11:09,266][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-stack]
[2024-11-11T21:11:09,266][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [rank-eval]
[2024-11-11T21:11:09,266][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [reindex]
[2024-11-11T21:11:09,266][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [test-apm-integration]
[2024-11-11T21:11:09,266][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-security]
[2024-11-11T21:11:09,267][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [blob-cache]
[2024-11-11T21:11:09,267][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [searchable-snapshots]
[2024-11-11T21:11:09,267][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-slm]
[2024-11-11T21:11:09,267][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-geoip-enterprise-downloader]
[2024-11-11T21:11:09,268][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [test-latency-simulating-directory]
[2024-11-11T21:11:09,268][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [snapshot-based-recoveries]
[2024-11-11T21:11:09,268][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-watcher]
[2024-11-11T21:11:09,268][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [old-lucene-versions]
[2024-11-11T21:11:09,268][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-ilm]
[2024-11-11T21:11:09,269][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-inference]
[2024-11-11T21:11:09,269][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-voting-only-node]
[2024-11-11T21:11:09,269][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-deprecation]
[2024-11-11T21:11:09,269][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-fleet]
[2024-11-11T21:11:09,270][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-aggregate-metric]
[2024-11-11T21:11:09,270][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-downsample]
[2024-11-11T21:11:09,270][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-profiling]
[2024-11-11T21:11:09,270][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [ingest-geoip]
[2024-11-11T21:11:09,271][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-write-load-forecaster]
[2024-11-11T21:11:09,271][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [search-business-rules]
[2024-11-11T21:11:09,271][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [ingest-attachment]
[2024-11-11T21:11:09,271][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [wildcard]
[2024-11-11T21:11:09,272][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-apm-data]
[2024-11-11T21:11:09,272][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [unsigned-long]
[2024-11-11T21:11:09,272][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-sql]
[2024-11-11T21:11:09,272][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [runtime-fields-common]
[2024-11-11T21:11:09,272][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-async]
[2024-11-11T21:11:09,273][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-kql]
[2024-11-11T21:11:09,273][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [vector-tile]
[2024-11-11T21:11:09,273][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [lang-expression]
[2024-11-11T21:11:09,273][INFO ][o.e.p.PluginsService     ] [bk-agent-prod-gcp-1731359030615088306] loaded module [x-pack-eql]
[2024-11-11T21:11:09,762][INFO ][o.e.c.u.FeatureFlag      ] [bk-agent-prod-gcp-1731359030615088306] The current build is a snapshot, feature flag [failure_store] is enabled
[2024-11-11T21:11:11,029][INFO ][o.e.e.NodeEnvironment    ] [bk-agent-prod-gcp-1731359030615088306] using [1] data paths, mounts [[/ (/dev/sda2)]], net usable_space [274.6gb], net total_space [349.7gb], types [xfs]
[2024-11-11T21:11:11,030][INFO ][o.e.e.NodeEnvironment    ] [bk-agent-prod-gcp-1731359030615088306] heap size [1gb], compressed ordinary object pointers [true]
[2024-11-11T21:11:11,041][INFO ][o.e.n.Node               ] [bk-agent-prod-gcp-1731359030615088306] node name [bk-agent-prod-gcp-1731359030615088306], node ID [vsfRtelMQ7Ko8CmKnWIySw], cluster name [elasticsearch], roles [data_frozen, ml, data_hot, transform, data_content, data_warm, master, remote_cluster_client, data, data_cold, ingest]
[2024-11-11T21:11:16,077][INFO ][o.e.c.u.FeatureFlag      ] [bk-agent-prod-gcp-1731359030615088306] The current build is a snapshot, feature flag [connector_secrets] is enabled
[2024-11-11T21:11:16,287][INFO ][o.e.i.r.RecoverySettings ] [bk-agent-prod-gcp-1731359030615088306] using rate limit [40mb] with [default=40mb, read=0b, write=0b, max=0b]
[2024-11-11T21:11:16,397][INFO ][o.e.c.u.FeatureFlag      ] [bk-agent-prod-gcp-1731359030615088306] The current build is a snapshot, feature flag [multi_dense_vector] is enabled
[2024-11-11T21:11:16,703][INFO ][o.e.c.u.FeatureFlag      ] [bk-agent-prod-gcp-1731359030615088306] The current build is a snapshot, feature flag [sub_objects_auto] is enabled
[2024-11-11T21:11:16,777][INFO ][o.e.c.u.FeatureFlag      ] [bk-agent-prod-gcp-1731359030615088306] The current build is a snapshot, feature flag [inference_default_elser] is enabled
[2024-11-11T21:11:16,781][INFO ][o.e.f.FeatureService     ] [bk-agent-prod-gcp-1731359030615088306] Registered local node features [cluster.reroute.ignores_metric_param, data_stream.auto_sharding, data_stream.lifecycle.global_retention, data_stream.rollover.lazy, desired_node.version_deprecated, esql.agg_values, esql.async_query, esql.base64_decode_encode, esql.casting_operator, esql.counter_types, esql.disable_nullable_opts, esql.from_options, esql.metadata_fields, esql.metrics_counter_fields, esql.metrics_syntax, esql.mv_ordering_sorted_ascending, esql.mv_sort, esql.resolve_fields_api, esql.spatial_points_from_source, esql.spatial_shapes, esql.st_centroid_agg, esql.st_contains_within, esql.st_disjoint, esql.st_intersects, esql.st_x_y, esql.string_literal_auto_casting, esql.string_literal_auto_casting_extended, esql.timespan_abbreviations, features_supported, file_settings, flattened.ignore_above_support, geoip.downloader.database.configuration, get_database_configuration_action.multi_node, health.dsl.info, health.extended_repository_indicator, knn_retriever_supported, license-trial-independent-version, logsdb_telemetry, logsdb_telemetry_stats, lucene_10_upgrade, mapper.boolean_dimension, mapper.bwc_workaround_9_0, mapper.flattened.ignore_above_with_arrays_support, mapper.ignore_above_index_level_setting, mapper.index_sorting_on_nested, mapper.keyword_dimension_ignore_above, mapper.keyword_normalizer_synthetic_source, mapper.pass_through_priority, mapper.query_index_mode, mapper.range.null_values_off_by_one_fix, mapper.segment_level_fields_stats, mapper.source.synthetic_source_copy_to_fix, mapper.source.synthetic_source_copy_to_inside_objects_fix, mapper.source.synthetic_source_fallback, mapper.source.synthetic_source_stored_fields_advance_fix, mapper.source.synthetic_source_with_copy_to_and_doc_values_false, mapper.subobjects_auto, mapper.subobjects_auto_fixes, mapper.synthetic_source_keep, mapper.track_ignored_source, mapper.vectors.bbq, mapper.vectors.bit_vectors, mapper.vectors.int4_quantization, put_database_configuration_action.ipinfo, query_rule_retriever_supported, query_rules.test, random_reranker_retriever_supported, repositories.supports_usage_stats, rest.capabilities_action, rest.local_only_capabilities, retrievers_supported, routing.boolean_routing_path, routing.multi_value_routing_path, rrf_retriever_composition_supported, rrf_retriever_supported, script.hamming, script.term_stats, search.vectors.k_param_supported, security.migration_framework, security.role_mapping_cleanup, security.roles_metadata_flattened, semantic_text.default_elser_2, semantic_text.inner_hits, semantic_text.search_inference_id, simulate.component.template.substitutions, simulate.index.template.substitutions, simulate.mapping.addition, simulate.mapping.validation, simulate.mapping.validation.templates, simulate.support.non.template.mapping, slm.interval_schedule, snapshot.repository_verify_integrity, standard_retriever_supported, stats.include_disk_thresholds, text_similarity_reranker_retriever_composition_supported, text_similarity_reranker_retriever_supported, tsdb.ts_routing_hash_doc_value_parse_byte_ref, unified_highlighter_matched_fields, usage.data_tiers.precalculate_stats]
[2024-11-11T21:11:16,830][INFO ][o.e.c.m.DataStreamGlobalRetentionSettings] [bk-agent-prod-gcp-1731359030615088306] Updated default factory retention to [null]
[2024-11-11T21:11:16,831][INFO ][o.e.c.m.DataStreamGlobalRetentionSettings] [bk-agent-prod-gcp-1731359030615088306] Updated max factory retention to [null]
[2024-11-11T21:11:27,711][ERROR][o.e.b.Elasticsearch      ] [bk-agent-prod-gcp-1731359030615088306] fatal exception while booting Elasticsearch org.elasticsearch.ElasticsearchException: Failure running machine learning native code. This could be due to running on an unsupported OS or distribution, missing OS libraries, or a problem with the temp directory. To bypass this problem by running Elasticsearch without machine learning functionality set [xpack.ml.enabled: false].
	at org.elasticsearch.ml@9.0.0-SNAPSHOT/org.elasticsearch.xpack.ml.MachineLearning.createComponents(MachineLearning.java:1061)
	at org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.node.NodeConstruction.lambda$construct$16(NodeConstruction.java:888)
	at org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.plugins.PluginsService.lambda$flatMap$1(PluginsService.java:254)
	at java.base/java.util.stream.ReferencePipeline$7$1FlatMap.accept(ReferencePipeline.java:289)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:722)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:570)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:636)
	at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:291)
	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:656)
	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:662)
	at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:667)
	at org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.node.NodeConstruction.construct(NodeConstruction.java:910)
	at org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.node.NodeConstruction.prepareConstruction(NodeConstruction.java:290)
	at org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.node.Node.<init>(Node.java:184)
	at org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.bootstrap.Elasticsearch$2.<init>(Elasticsearch.java:245)
	at org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.bootstrap.Elasticsearch.initPhase3(Elasticsearch.java:245)
	at org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:76)

ERROR: Elasticsearch did not exit normally - check the logs at /tmp/elasticsearch/logs/elasticsearch.log

ERROR: Elasticsearch died while starting up, with exit code 1

Failed to determine if startup succeeded] stderr = []]
Expected: <0>
     but: was <1>
```

**Issue Reasons:**
- [main] 2 consecutive failures in test test20NoAutoGenerationWhenAutoConfigurationDisabled
- [main] 22 failures in test test20NoAutoGenerationWhenAutoConfigurationDisabled (3.1% fail rate in 710 executions)
- [main] 9 failures in step rhel-8_destructivedistrotest.archives_packaging-tests-unix-sample (3.8% fail rate in 234 executions)
- [main] 9 failures in step ubuntu-2004_destructivedistrotest.archives_packaging-tests-unix-sample (3.9% fail rate in 233 executions)
- [main] 9 failures in pipeline elasticsearch-pull-request (3.8% fail rate in 237 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -224,15 +224,6 @@ tests:
 - class: org.elasticsearch.upgrades.SearchStatesIT
   method: testCanMatch
   issue: https://github.com/elastic/elasticsearch/issues/116618
-- class: org.elasticsearch.packaging.test.ArchiveGenerateInitialCredentialsTests
-  method: test20NoAutoGenerationWhenAutoConfigurationDisabled
-  issue: https://github.com/elastic/elasticsearch/issues/116619
-- class: org.elasticsearch.packaging.test.BootstrapCheckTests
-  method: test20RunWithBootstrapChecks
-  issue: https://github.com/elastic/elasticsearch/issues/116620
-- class: org.elasticsearch.packaging.test.DockerTests
-  method: test011SecurityEnabledStatus
-  issue: https://github.com/elastic/elasticsearch/issues/116628
 - class: org.elasticsearch.reservedstate.service.RepositoriesFileSettingsIT
   method: testSettingsApplied
   issue: https://github.com/elastic/elasticsearch/issues/116694

